packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -68,10 +69,10 @@ export const filterSettingsToggleLabels = forumSelect({
     mobileHidden: "Customize feed",
   },
   default: {
-    desktopVisible: "Customize Feed (Hide)",
-    desktopHidden: "Customize Feed",
-    mobileVisible: "Customize Feed (Hide)",
-    mobileHidden: "Customize Feed (Show)",
+    desktopVisible: "Customize (Hide)",
+    desktopHidden: "Customize",
+    mobileVisible: "Customize (Hide)",
+    mobileHidden: "Customize",
```

Comment:
This will change the button text for all users on LW, not just admins or those who pass `userHasRecombeeFrontpage`

Reply:
I know. I originally labeled the buttons and now feel this is better.